src/librustc/middle/stability.rs

```diff
@@ -313,8 +313,9 @@ struct MissingStabilityAnnotations<'a, 'tcx: 'a> {
 impl<'a, 'tcx: 'a> MissingStabilityAnnotations<'a, 'tcx> {
     fn check_missing_stability(&self, id: NodeId, span: Span) {
         let def_id = self.tcx.hir.local_def_id(id);
+        let stab = self.tcx.stability.borrow().stab_map.get(&def_id).cloned();
```

Comment:
While I doubt this has any serious implications, the `cloned` here seems unnecessary since AFAICT we never use anything but (one of) the `None` variants inside. Could we remove it?